src/ui/flamegraph.rs

```diff
@@ -110,4 +114,22 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
```

Comment:
I like this. I think it would be nice to have a test for the contents of the collapsed file as well.